wikidatarefisland/pipes/pipe_one.py

```diff
@@ -0,0 +1,47 @@
+from itertools import chain
+from wikidatarefisland.pipes import statement_filters
+from .abstract_pipe import AbstractPipe
+from wikidatarefisland.data_model import statement_filterer
+
+
+class ItemExtractorPipe(AbstractPipe):
+    def __init__(self, external_id_formatter, storage,
+                 blacklisted_properties=None):
+        self._storage = storage
+        if blacklisted_properties is None:
+            blacklisted_properties = []
+        self.blacklisted_properties = blacklisted_properties
+        self.external_id_formatter = external_id_formatter
+
+    def flow(self, input_data):
+        return self._process_item(input_data)
+
+    def _process_item(self, item_data):
```

Comment:
This doesn't skip items that are scientific paper, galaxy, and so on: https://github.com/wmde/reference-island/blob/master/config/default.yml#L1